resources/templates/md/posts/2016-04-04-state-management.md

```diff
@@ -0,0 +1,364 @@
+{:title "Clojure state management"
+:description  "How we can manage states in Clojure (how data might change over time), or llamas love candies too :)"
+:layout :post
+:tags  ["Clojure" "ClojureScript" "State management"]
+:toc false
+:author "Dajana Štiberová"
+:image ""}
+
+Long time ago, during the one interview, I was asked to describe the difference between `vars` and `atoms`.
+It was pretty simple question, but at that time it was really hard to describe this difference for me.
+Now I'm more skilled in Clojure but I'm still thinking that this is an interesting question and when
+I was looking for some topic for the blog post, I decided to use this one.
+
+So if you want to read something about state management, or you want to know how to dance like Greg the llama, this post is for you :)
+
+## States and Identities
+
+We need to know that in Clojure, there is the difference between the state and the identity.
+
+`States` are an immutable data.
+A state is simply the value of an identity at a particular time. State never changes.
+
+`Identities` unify different values over the time.
+Identity types are mutable references to immutable values,
+and can be modeled by using a reference types (`Vars`, `Atoms`, `Agents`, `Refs`)
```

Comment:
by using one of the four references types: `vars`, `atom`s, `agent`s and `ref`s.